src/index.html

```diff
@@ -6,8 +6,51 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="APPLE_A1419_iMac_27"
+        class="card__image"
+      >
+
+      <h1 class="card__description">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+
+      <div class="card__code">
+        <p>Product code: 195434</p>
+      </div>
+
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p>Reviews: 5</p>
+      </div>
+
+      <h1 class="card__price">
```

Comment:
h1 should be one per page

anyway, this is not a suitable case to use h1 just inside a card. there can be a lot of cards, so choose another one
I'd recommend h2 as a minimum